tpu-client-next/src/send_transaction_stats.rs

```diff
@@ -5,13 +5,8 @@ use {
     super::QuicError,
     quinn::{ConnectError, ConnectionError, WriteError},
     std::{
-        collections::HashMap,
```

Comment:
It looks like documentation for this module is stale: we don't collect per IP statistics anylonger

Reply:
Fixed in https://github.com/anza-xyz/agave/pull/5338/commits/81ee8a11713786c521d1c45d65989aba2776db75